src/chains/ethereum/options/src/wallet-options.ts

```diff
@@ -229,7 +229,13 @@ export const WalletOptions: Definitions<WalletConfig> = {
     conflicts: ["totalAccounts"]
   },
   totalAccounts: {
-    normalize,
+    normalize: rawInput => {
+      if (rawInput >= MAX_ACCOUNTS)
+        console.log(
```

Comment:
this should probably be `console.warn`

Reply:
The validation should be moved to the wallet, where it would use logger.log

Reply:
@davidmurdoch are you sure? Don't we log other startup data to `console.log`?

Reply:
I'm pretty sure. Where else do we use `console.log`?

Reply:
When we start up the CLI: https://github.com/trufflesuite/ganache/blob/32a78143744b9191971f1c1bf029f7725d081abe/src/packages/cli/src/initialize/ethereum.ts#L129

Reply:
The user can't control the `logger` on the CLI.

Reply:
Note: we don't respect the "quiet" flag (which sets the logger to a noop) for the startup CLI output. So an argument could be made for this specific case that the warning should _also_ not respect the `quiet` flag. :thinking:

Reply:
Yes, @davidmurdoch, that was the argument I was attempting to make!

Reply:
I can always count on you to make my arguments more intelligible :heart:

Reply:
But it was a trick; this message is not CLI-only output. :-p This log output would also appear in programmatic use, so I think we should respect their user's logger (or quiet flag) 🤔

Reply:
@tenthirtyone do what that guy (^) says, he seems smart